moto/elbv2/models.py

```diff
@@ -214,30 +215,23 @@ def __init__(
         self.certificate = certificate
         self.certificates = [certificate] if certificate is not None else []
         self.default_actions = default_actions
-        self._non_default_rules = []
-        self._default_rule = FakeRule(
```

Comment:
What's the reasoning behind removing the default rule? If I understand the docs correctly, the default rule should be created automatically when creating a listener, regardless of any rules created by the user

Reply:
You are correct. My bad. Fixed: ffd63ba